tests/components/rfxtrx/test_sensor.py

```diff
@@ -86,27 +85,23 @@ async def test_several_sensors(hass, rfxtrx):
     )
     await hass.async_block_till_done()
 
-    assert 2 == len(rfxtrx_core.RFX_DEVICES)
-    device_num = 0
-    for id in rfxtrx_core.RFX_DEVICES:
-        if id == "sensor_06_01":
-            device_num = device_num + 1
-            assert len(rfxtrx_core.RFX_DEVICES[id]) == 2
-            _entity_temp = rfxtrx_core.RFX_DEVICES[id]["Temperature"]
-            _entity_hum = rfxtrx_core.RFX_DEVICES[id]["Humidity"]
-            assert UNIT_PERCENTAGE == _entity_hum.unit_of_measurement
-            assert "Bath" == _entity_hum.__str__()
-            assert _entity_hum.state is None
-            assert TEMP_CELSIUS == _entity_temp.unit_of_measurement
-            assert "Bath" == _entity_temp.__str__()
-        elif id == "sensor_05_02":
-            device_num = device_num + 1
-            entity = rfxtrx_core.RFX_DEVICES[id]["Temperature"]
-            assert entity.state is None
-            assert TEMP_CELSIUS == entity.unit_of_measurement
-            assert "Test" == entity.__str__()
-
-    assert 2 == device_num
+    state = hass.states.get("sensor.test_temperature")
+    assert state
+    assert state.state == "unknown"
+    assert state.attributes.get("friendly_name") == "Test Temperature"
+    assert state.attributes.get("unit_of_measurement") == TEMP_CELSIUS
+
+    state = hass.states.get("sensor.bath_temperature")
+    assert state
+    assert state.state == "unknown"
+    assert state.attributes.get("friendly_name") == "Bath Temperature"
+    assert state.attributes.get("unit_of_measurement") == TEMP_CELSIUS
+
+    state = hass.states.get("sensor.bath_humidity")
+    assert state
+    assert state.state == "unknown"
+    assert state.attributes.get("friendly_name") == "Bath Humidity"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)